src/main/kotlin/com/vk/admstorm/services/SentryService.kt

```diff
@@ -29,14 +30,25 @@ class SentryService(project: Project) {
     private val user = GitUtils.currentUser(project)
 
     init {
+        initSentry(project)
+    }
+
+    private fun initSentry(project: Project) {
+        if (ApplicationManager.getApplication().isInternal) {
+            LOG.info("Sending errors to Sentry is disabled. The IDE is running in development mode")
+            return
+        }
+
         val config = ConfigService.getInstance(project)
         val plugin = PluginManagerCore.getPlugin(AdmService.PLUGIN_ID)
         val application = ApplicationInfo.getInstance()
 
         if (config.sentryDsn.isEmpty()) {
```

Comment:
Can we move this condition after `config` and before `plugin` variables?

Reply:
fixed